test/RunCi.hx

```diff
@@ -385,7 +385,7 @@ Listen 2000
 			"-D",
 			'haxelib_path=${Path.join([Sys.getCwd(), "haxelib"])}'
 		]);
-		runCommand("neko", ["bin/integration_tests.n"]);
+		// runCommand("neko", ["bin/integration_tests.n"]);
```

Comment:
Is this supposed to be disabled now?

Reply:
Yes they should be for now, it is explained in the commit message of 824c06d080aa885021273acc33af574e26a539a8. This test run uses the cpp binary as the haxelib client (specified by `haxelib_path`), but that is currently broken. The test with the regular neko client still runs on the lines above.